spring-boot-examples/workflows/src/main/java/io/dapr/springboot/examples/wfp/WorkflowPatternsConfiguration.java

```diff
@@ -13,14 +13,27 @@
 
 package io.dapr.springboot.examples.wfp;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.dapr.springboot.examples.wfp.continueasnew.CleanUpLog;
+import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
 
 @Configuration
 public class WorkflowPatternsConfiguration {
   @Bean
   public CleanUpLog cleanUpLog(){
     return new CleanUpLog();
   }
+
```

Comment:
I am wondering if we should use the recommended `RestClient` instead of `RestTemplate` as far as I know `RestTemplate` has been deprecated.